apps/bank-connect-clone/src/app/dashboard/dashboard.component.scss

```diff
@@ -0,0 +1,31 @@
+mat-sidenav {
+  background-color: #f2f3f7 !important;
+}
+
+.icici-logo-color {
+  background-color: rgb(242, 243, 247) !important;
+}
+.acc-details {
+  background-color: rgb(238, 240, 246);
+}
+.image-container {
+  position: relative;
+}
+
+.hover-img {
+  display: none;
+}
+
```

Comment:
can you make this style in tailwind class itself or if it not required remove this style